docs/src/content/mapping/lodash/debounce.md

```diff
@@ -1,4 +1,137 @@
 ---
 category: Function
-remeda: debounce
+remeda: funnel
 ---
+
+- `debounce` can be implemented using the `funnel` utility. A reference
+  implementation is provided below, and a more expanded version with inline
+  documentation and tests is available in the test file [`funnel.lodash-debounce.test.ts`](https://github.com/remeda/remeda/blob/main/src/funnel.lodash-debounce.test.ts).
+
+- A more complete reference implementation that also maintains Lodash's
+  capability to store the callback's return value is available below, and in [`funnel.lodash-debounce-with-cached-value.test.ts`](https://github.com/remeda/remeda/blob/main/src/funnel.lodash-debounce-with-cached-value.test.ts).
+
+- These implementations can be copied as-is into your project, but might contain
+  redundant parts which are not relevant for your specific use cases. By
+  inlining only the parts you need you can take advantage of capabilities not
+  available in Lodash.
+
+### Reference
```

Comment:
**thought:** maybe we can collect these in the docs under a `remeda-contrib` thing...? hm

Reply:
there's libraries called "lodash-contrib" and "underscore-contrib".

i actually don't like my idea anymore